app/main/src/main.ts

```diff
@@ -73,7 +73,7 @@ function createMainWindow() {
 	win = new BrowserWindow({
 		width: 340,
 		height: getFrameHeight(),
-		resizable: true,
+		resizable: false,
```

Comment:
If its only being set via code this shouldn't be true ever. It seems altering between true and false makes mac confused for full sizing.